library/pp/mcall_spec.rb

```diff
@@ -0,0 +1,6 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require 'pp'
+
+describe "PP.mcall" do
```

Comment:
Sure. That was the generator. I saw some other specs that had the skeletons still there, so I followed that, but will be happy to remove these.